html/semantics/forms/form-submission-0/constructing-form-data-set.html

```diff
@@ -46,4 +47,64 @@
   form2.submit.click();
 }, 'The button cannot be setted if it is not a submitter.');
 
+test(() => {
+  let didCallHandler = false;
+  let wasBubbles = false;
+  let wasCancelable = true;
+  let form = populateForm();
+  document.addEventListener('formdata', e => {
+    didCallHandler = true;
+    wasBubbles = e.bubbles;
+    wasCancelable = e.cancelable;
+  });
+  new FormData(form);
+  assert_true(didCallHandler);
+  assert_true(wasBubbles);
```

Comment:
@tkent-google shouldn't this be `assert_false`? Per the spec, it doesn't bubble. Or did I miss that somewhere?

Reply:
Oops, it's a spec bug.  Please trust the test.
I'll make a specification PR.

Reply:
PR: https://github.com/whatwg/html/pull/4422